phac-nml/irida-next
# Group Advanced Settings - Transfer

## What does this PR do and why?
Created a transfer group section on the group settings page. Fixes the second half of https://github.com/phac-nml/irida-next/issues/75.

## Screenshots or screen recordings
![image](https://github.com/phac-nml/irida-next/assets/325703/d6fd3ab1-a828-4772-95f1-254ff010602e)
![image](https://github.com/phac-nml/irida-next/assets/325703/824c3feb-ad98-4316-8ba7-f57cdf7373d0)

## How to set up and validate locally
1. Navigate to a group that is owned by the logged-in user.
2. Hit `Settings` on the left side menu.
3. Under `Advanced settings` you will see a `Transfer Group` section. Select a new parent group from the drop-down.
4. Hit the `Transfer group` button.
5. A confirmation dialog should appear. Type the group path and hit `Confirm`.

## PR acceptance checklist
This checklist encourages us to confirm any changes have been analyzed to reduce risks in quality, performance, reliability, security, and maintainability.

  the [PR acceptance checklist](https://phac-nml.github.io/irida-next/docs/development/development_processes/code_review#acceptance-checklist) for this PR.
